util/unittests/test_html.py

```diff
@@ -283,15 +283,15 @@ def setUpClass(cls):
         cls.map_dict['result3'] = 'switches'
         cls.map_dict['result4'] = 'switches'
 
-    def test01_get_ordered_index(self):
+    def test_get_ordered_index(self):
         """Method that tests functionality of get_ordered_index() function
         in case where it returns a valid index"""
         self.assertEqual(1,
                          util.html.get_ordered_index('result3',
                                                      self.map_dict),
                          'Testing returned index when key exists.')
 
-    def test02_get_ordered_index(self):
+    def test_get_ordered_index(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_get_ordered_index' from line 286

<!-- Comment by [SideCI](https://sideci.com) -->